test/cqlpy/test_decimal.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+#############################################################################
+# Test involving the "decimal" column type.
+# There are additional tests involving decimals in specific contexts in other
+# files - e.g., aggregating decimals in test_aggregate.py, casting decimals
+# in test_cast_data.py, and decimals in JSON in test_json.py
+#############################################################################
```

Comment:
Please consider using for this file the name test_type_decimal.py, like I suggested, and not test_decimal. I have other files `test_type_*` (date, duration, string, time, timestamp, uuid). I think it's good to know that these tests are about the CQL type "decimal" and not other things which might be called "decimal" (for type_timestamp it's even more obvious that it's an important distinction).

Reply:
Done in v4.1.